assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -653,7 +562,9 @@ function initialize(api) {
       return new RenderGlimmer(
         this,
         "p.assigned-to",
-        hbs`<AssignedToPost @assignedToUser={{@data.assignedToUser}} @assignedToGroup={{@data.assignedToGroup}} @href={{@data.href}} @post={{@data.post}} />`,
+        hbs`
+          <AssignedToPost @assignedToUser={{@data.assignedToUser}} @assignedToGroup={{@data.assignedToGroup}}
+                          @href={{@data.href}} @post={{@data.post}} />`,
```

Comment:
formating seems odd, but maybe it's github

Reply:
It wasn't GitHub, there were an accident change here, it's fixed now.